components/HomepageSection/PropelledBy.jsx

```diff
@@ -23,7 +27,11 @@ const PropelledBy = () => (
           } = builder;
           return (
             <div key={id} className="grayscale flex justify-center items-center">
-              <Link href={url}>
+              <Link
+                href={url}
+                target="_blank"
```

Comment:
made these links external, I hope no one minds, it seems wrong to take the users away from the site

Reply:
good point, you're right. I'll update that in the next PR as I already used `Link` with no profit in other places in previous PR.